scripts/monitor_blocks.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
We got some stats for this now

Reply:
Can we leave it in?
I've been using it on prod to make sure router is catching back up (metrics haven't been deployed yet).
And it could be useful for open source users who don't hook hook up metrics.

Reply:
Ok seems fair